src/v2/Apps/Order/Components/ArtworkSummaryItem.tsx

```diff
@@ -46,31 +44,31 @@ const ArtworkSummaryItem: React.SFC<ArtworkSummaryItemProps> = ({
   return (
     <StackableBorderBox flexDirection="row" {...others}>
       <Box height="auto">
-        {imageURL && <Image src={imageURL} alt={title} width="55px" mr={1} />}
+        {imageURL && <Image src={imageURL} alt={title!} width="55px" mr={1} />}
       </Box>
-      <Flex flexDirection="column" style={{ overflow: "hidden" }}>
-        <Serif
-          size="2"
-          weight="semibold"
-          color="black60"
-          style={truncateTextStyle}
-        >
+      <Flex flexDirection="column" overflow="hidden">
+        <Text variant="sm" color="black60">
           {artist_names}
-        </Serif>
-        <div style={{ lineHeight: "1", ...truncateTextStyle }}>
-          <Serif italic size="2" color="black60" display="inline">
+        </Text>
+        <Box style={{ lineHeight: "1", ...truncateTextStyle }}>
+          <Text
+            fontStyle="italic"
+            variant="sm"
+            color="black60"
+            display="inline"
+          >
             {title}
-          </Serif>
-          <Serif size="2" color="black60" display="inline">
+          </Text>
+          <Text variant="sm" color="black60" display="inline">
             {date && `, ${date}`}
-          </Serif>
-        </div>
-        <Serif size="2" color="black60" style={truncateTextStyle}>
+          </Text>
+        </Box>
+        <Text variant="sm" color="black60" style={truncateTextStyle}>
```

Comment:
You can delete these `truncateTextStyle` and just use the `overflowEllipsis` prop